core/test/integration/helpers/ghost_head_spec.js

```diff
@@ -1218,6 +1218,19 @@ describe('{{ghost_head}} helper', function () {
             }).catch(done);
         });
 
+        it('disallows indexing for preview pages', function (done) {
```

Comment:
So this test makes sense, because we want to make sure the tag is added to `/p/` pages.

How should I go about adding tests to make sure it's not added to other pages? Should I add `should.not.match` lines to other tests where the `relativeUrl` parameter is (semi-arbitrarily) distinct?

Reply:
As far as tests go, I'm not certain due to the complexities of the Ghost ecosystem, but [maybe this](https://github.com/TryGhost/Ghost/pull/9762/files#r207703580) will help.

Reply:
Adding one assertion to proof that the meta-tag is not present when `preview` is not included in the context object is enough.